share/jupyterhub/templates/not_running.html

```diff
@@ -58,11 +58,7 @@ <h1>
   {% if implicit_spawn_seconds %}
     <script type="text/javascript">
       var spawn_url = "{{ spawn_url }}";
-      var implicit_spawn_seconds = {
-        {
-          implicit_spawn_seconds
-        }
-      };
+      var implicit_spawn_seconds = {{ implicit_spawn_seconds }};
```

Comment:
It seems like that since this was not put in quotes, the linter reacted on this but not on `spawn_url`.